Packs/FireEyeCM/Integrations/FireEyeCM/FireEyeCM_test.py

```diff
@@ -345,15 +345,18 @@ def test_fetch_incidents_last_alert_ids(mocker):
     - mock the Client's get_alerts_request.
     Then
     - Validate that no incidents will be returned.
-    - Validate that the last_run is at the latest incident fetched
+    - Validate that the last_run is pushed in two days from the latest incident fetched
```

Comment:
do we have test where can check that `now - 10minutes` is taken?

Reply:
Added